modules/nux/site.js

```diff
@@ -57,7 +57,7 @@ var display_final_nux_step = function(res) {
                 }
             );
         } else {
-            Hm_Utils.redirect();
+            Hm_Utils.redirect('?page=servers');
```

Comment:
at https://github.com/cypht-org/cypht/blob/master/modules/core/site.js#L1711 we have window.location.href that is already redirecting to the current URL of the page that is being displayed in the browser, in this case it will be `www.domain.com?page= servers`, even if we did not specify this parameter.

Have you had a case? I can't reproduce a one that redirects to an another page than `servers`.

Reply:
Hi @Shadow243,
You’re right! After updating to the latest version of the master branch, I’m no longer able to reproduce the bug. Let me test various cases to ensure it's completely fixed.

Reply:
I can not reproduce this too... I have tried several time. @henochit have you tried too so we know...

Reply:
Isn't it related to the no redirect while adding an account from home page quick add section.... I got this:
<img width="961" alt="servers_quick_add_section_from_home_page" src="https://github.com/user-attachments/assets/640c00f7-25ee-485c-a418-ee26880f7f45">
@henochit ?